src/OSPSuite.Core/Converter/v5_2/DimensionMapper.cs

```diff
@@ -35,6 +35,8 @@ public interface IDimensionMapper
 
    internal class DimensionMapper : IDimensionMapper
    {
+      private const string RHS_DIMENSION_SUFFIX = " per Time";
```

Comment:
@Yuri05 I would vote to remove all those conversions here. Trying to convert pkml from version 52 becomes ridiculous
I think we should support starting 6.3.1